### PR TITLE
Make paren-mismatch more obvious and add new faces

### PR DESCRIPTION
**Pull request by nnoodle**

### CODE REVIEW

File: monokai-pro-theme.el [hunk 1]
```diff
@@ -367,19 +378,20 @@
...
+     (rainbow-delimiters-depth-1-face              :inherit rainbow-delimiters-base-face :foreground red)
+     (rainbow-delimiters-depth-2-face              :inherit rainbow-delimiters-base-face :foreground pink)
+     (rainbow-delimiters-depth-3-face              :inherit rainbow-delimiters-base-face :foreground orange)
+     (rainbow-delimiters-depth-4-face              :inherit rainbow-delimiters-base-face :foreground yellow)
+     (rainbow-delimiters-depth-5-face              :inherit rainbow-delimiters-base-face :foreground green)
+     (rainbow-delimiters-depth-6-face              :inherit rainbow-delimiters-base-face :foreground blue)
+     (rainbow-delimiters-depth-7-face              :inherit rainbow-delimiters-base-face :foreground purple)
+     (rainbow-delimiters-depth-8-face              :inherit rainbow-delimiters-base-face :foreground fg-1)
+     (rainbow-delimiters-depth-9-face              :inherit rainbow-delimiters-base-face :foreground fg)
+     (rainbow-delimiters-unmatched-face            :inherit (rainbow-delimiters-base-face show-paren-mismatch))
```
Comment (belak) [inline monokai-pro-theme.el]: I think there might be a mistake on this line

**Review by belak — COMMENTED** [review]
Is there any chance you have a before/after photo? That makes it way easier to merge changes.

**Comment by belak** [comment]
Thanks for submitting this by the way!

**Comment by nnoodle** [comment]
> Is there any chance you have a before/after photo?

Yeah sure, I think that'll make what I'm talking about clearer.

Before it was difficult to see where the mismatch parenthesis is, and where the currently matched parentheses even is.
before
This PR makes the mismatch more obvious, and make the matched parentheses a bit clearer to notice.
after

File: monokai-pro-theme.el (hunk 1 above)
Comment (nnoodle) [reply monokai-pro-theme.el]: Ah yeah you're right, `show-paren-mismatch` should have had higher priority over the rainbow delimiters base face, I reversed the order in a new commit.

**Comment by belak** [comment]
This is a pretty huge usability improvement. Thanks for submitting this!

**Pull request merged**